### ISSUE TITLE
Extra bridge interfaces in nat iptables

### ISSUE THREAD

**Issue by sburnwal** [docker/docker#30233]
I see 3 bridges in my nat iptables for the same bridge network (169.254.1.0/24) which I had created using docker network create. Can someone explain me why do I see 3 interfaces under POSTROUTING chain ? When I run 'ifconfig' command on my linux box, I see only br-7b745bd328f is listed.

```
[root@mylinux] iptables -nvL -t nat
Chain PREROUTING (policy ACCEPT 2401 packets, 223K bytes)
 pkts bytes target     prot opt in     out     source               destination         
 1146  123K DOCKER     all  --  *      *       0.0.0.0/0            0.0.0.0/0            ADDRTYPE match dst-type LOCAL

Chain INPUT (policy ACCEPT 1146 packets, 123K bytes)
 pkts bytes target     prot opt in     out     source               destination         

Chain OUTPUT (policy ACCEPT 6607 packets, 532K bytes)
 pkts bytes target     prot opt in     out     source               destination         
 1125  128K DOCKER     all  --  *      *       0.0.0.0/0           !127.0.0.0/8          ADDRTYPE match dst-type LOCAL

Chain POSTROUTING (policy ACCEPT 6587 packets, 531K bytes)
 pkts bytes target     prot opt in     out     source               destination         
   30  1933 MASQUERADE  all  --  *      !br-7b745bd328fa  169.254.1.0/24       0.0.0.0/0           
   20  1200 MASQUERADE  all  --  *      !br-a74636275be3  169.254.1.0/24       0.0.0.0/0           
    0     0 MASQUERADE  all  --  *      !br-3d457e90c43c  169.254.1.0/24       0.0.0.0/0           
    0     0 MASQUERADE  all  --  *      !docker0  172.17.0.0/16        0.0.0.0/0           
    0     0 MASQUERADE  tcp  --  *      *       169.254.1.4          169.254.1.4          tcp dpt:3000

Chain DOCKER (2 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DNAT       tcp  --  !br-7b745bd328fa *       0.0.0.0/0            127.0.0.1            tcp dpt:3000 to:169.254.1.4:3000

```

**Comment by sburnwal** [comment]
This is with version 1.9.1

**Comment by fntlnz** [comment]
can you do a `brctl show` on the host?

**Comment by aboch** [comment]
@sburnwal 
They are likely stale entries from previous daemon run. We resolved this issue in subsequent releases.

**Comment by aboch** [comment]
@sburnwal 
I found the original issue is a duplicate of #18169, and was fixed by docker/libnetwork#591
The fix is available starting from docker 1.10.0

I will go ahead and close this one.
